examples/lora/sensor-receive/Makefile

```diff
@@ -0,0 +1,14 @@
+# Makefile for user application
+
+# Specify this directory relative to the current application.
+TOCK_USERLAND_BASE_DIR = ../../..
+
+# Which files to compile.
+CXX_SRCS := $(wildcard *.cc)
+
+# Use the libtock-c Make system
+include $(TOCK_USERLAND_BASE_DIR)/libradio/Makefile
+
+include $(TOCK_USERLAND_BASE_DIR)/libradio/Makefile.app
```

Comment:
These should just set the `EXTERN_LIBS` variable instead of including Makefiles.

Reply:
That doesn't work

```
  LD        build/cortex-m0/cortex-m0.elf
/usr/lib/gcc/arm-none-eabi/14.1.0/../../../../arm-none-eabi/bin/ld: cannot find ../../../libradio/build/cortex-m0/libradio.a: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [../../../AppMakefile.mk:340: build/cortex-m0/cortex-m0.elf] Error 1
```

While the file is at `../../../libradio/RadioLib/build/cortex-m0/RadioLib.a`

Reply:
Fixed!